domains/emissions/chart/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: eo-emissions
 description: A chart containing the emissions domain.
 
 type: application
-version: 0.3.7
+version: 0.3.8
```

Comment:
I would argue these changes merit a major version change instead of a patch.

Reply:
bumped in 18708ff